src/components/ui/Session.tsx

```diff
@@ -0,0 +1,170 @@
+import { Modal, StyleSheet, Text, View } from "react-native";
```

Comment:
we could move this component into a folder called sessions 

src/components/sessions/SessionControl.tsx


I think that we could separate a bit more the Timer logic and the Session logic but it's okay

Reply:
Ok I moved the component to the suggested folder. Maybe I can separate the logic more later.